### ISSUE TITLE
Errors on RPi verilator 3.9 & 3.93

### ISSUE THREAD

**Issue by develone** [ZipCPU/xulalx25soc#1]
see the errors with verilator 3.9 & 3.93 https://gist.github.com/8c8d335ef153d1ca8e1e4900717ef8f0.git

**Comment by ZipCPU** [comment]
Could not duplicate: I'm not getting any errors with Verilator 3.874, Verilator 3.916, or Verilator git head.  I tried with both XULA25 set and commented.

These versions should also have the problems you are identifying in your gist.  Are you sure you have the right version of ZipBones?

**Comment by develone** [comment]
Dan the version from your repo xulalx25soc is  the correct my repo is not correct but even with yours on my new ubuntu I built busmaster_tb it starts but wbregs does not connect will into some more

Sent from Yahoo Mail on Android 
 
  On Fri, Oct 19, 2018 at 12:34 PM, Dan Gisselquist<notifications@github.com> wrote:   
Could not duplicate: I'm not getting any errors with Verilator 3.874, Verilator 3.916, or Verilator git head. I tried with both XULA25 set and commented.

These versions should also have the problems you are identifying in your gist. Are you sure you have the right version of ZipBones?
